### DISCUSSION TITLE
Is anyone having issues with Hydrogen suddenly?

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#22191] [General]
I guess my Atom self-updated today to 1.56.0, and now my Hydrogen (2.14.7) is experiencing rebuild failure. At first, I was getting an error related to Xcode, so I ran `xcode-select --install` in terminal, but now I am getting another error: `../binding.cc:28:10: fatal error: 'zmq.h' file not found`.

At this point, I'm not sure what to do. I am thinking the issue will resolve itself when Hydrogen eventually updates to work with the latest version of Atom, but in the interim, I was wondering if anyone can confirm if they are also experiencing the same issue, and if there is a solution?

**Comment by UziTech** [comment]
Have you tried uninstalling and reinstalling hydrogen? Any time Atom updates the version of node bundled with Atom packages with native modules must be rebuilt to match the new version of node. If something goes wrong during the rebuild process the easiest way to fix it is to uninstall and reinstall the package (sometimes a computer restart will also be necessary).

**Reply by ghost** [reply]
It worked! Thank you so much!

The full steps of what I did, for anyone with the same issue:
1.  Uninstall and reinstall Hydrogen
2. Completely close (quit) Atom and then reopen
3. Rebuild (should be successful now)

**Comment by ArthurHowardMorris** [comment]
I encountered the same issue (Atom: 1.56.0, Hydrogen: 2.14.7, Mac OS 11.2.3). But after uninstalling Hydrogen, install fails with the following warnings and errors: 

```
prebuild-install WARN install No prebuilt binaries found (target=9.4.4 runtime=electron arch=x64 libc= platform=darwin)
``` 

```
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated json-parser@1.1.5: `json-parser` is deprecated. Please use `comment-json` instead
npm WARN deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
configure: WARNING: Cannot find libunwind
```

```
gyp: No Xcode or CLT version detected!
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 20.3.0
gyp ERR! command "/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /private/var/folders/jr/5mb_36xd1pn6tr_bb5t6w3zc0000gn/T/apm-install-dir-2021315-7819-l6sygr.wgoe/node_modules/Hydrogen/node_modules/zeromq
gyp ERR! node -v v12.4.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open '/private/var/folders/jr/5mb_36xd1pn6tr_bb5t6w3zc0000gn/T/apm-install-dir-2021315-7819-l6sygr.wgoe/package.json'
npm WARN apm-install-dir-2021315-7819-l6sygr.wgoe No description
npm WARN apm-install-dir-2021315-7819-l6sygr.wgoe No repository field.
npm WARN apm-install-dir-2021315-7819-l6sygr.wgoe No README data
npm WARN apm-install-dir-2021315-7819-l6sygr.wgoe No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zeromq@5.2.4 install: `node scripts/prebuild-install.js || (node scripts/preinstall.js && node-gyp rebuild)`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zeromq@5.2.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by ptoche** [comment]
A new version has just been released, try installing now.

**Reply by ArthurHowardMorris** [reply]
Installing Hydrogen 2.15.2 solved the problem. Big thanks to the community behind atom and hydrogen!
